images/hub/requirements.txt

```diff
@@ -2,7 +2,7 @@
 # This file is autogenerated by pip-compile with python 3.9
 # To update, run:
 #
-#    Use the "Run workflow" button at https://github.com/jupyterhub/zero-to-jupyterhub-k8s/actions/workflows/watch-dependencies.yaml
+#    pip-compile
```

Comment:
```suggestion
# Use the "Run workflow" button at https://github.com/jupyterhub/zero-to-jupyterhub-k8s/actions/workflows/watch-dependencies.yaml
```

Reply:
Ah, I just went down the wrong track because of the dependabot mention which I knew wasn't right. I assumed that meant the comment was just on the wrong file, since it _does_ describe requirements.txt.  Sorry!